Timeout of 0.6 seconds elapsing.

macOS intel

` <PackageVersion Include="GitHub.Copilot.SDK" Version="0.1.26-preview.0" />`


/Users/admin/.nuget/packages/github.copilot.sdk/0.1.26-preview.0/build/GitHub.Copilot.SDK.targets(77,5): error MSB3923: Failed to download file "https://registry.npmjs.org/@github/copilot-darwin-x64/-/copilot-darwin-x64-0.0.411.tgz".  The request was canceled due to the configured HttpClient.Timeout of 0.6 seconds elapsing. ---> The operation was canceled. ---> The operation was canceled. ---> Unable to read data from the transport connection: Operation canceled. ---> Operation canceled [/Users/admin/Documents/Orchard_server/OC_Commerce_Modules/src/GuangCore/GuangCore.Utilities/GuangCore.Utilities.csproj]

--- a/dotnet/src/build/GitHub.Copilot.SDK.targets
+++ b/dotnet/src/build/GitHub.Copilot.SDK.targets
@@ -66,6 +66,8 @@
       <_CopilotArchivePath>$(_CopilotCacheDir)\copilot.tgz</_CopilotArchivePath>
       <_CopilotNormalizedRegistryUrl>$([System.String]::Copy('$(CopilotNpmRegistryUrl)').TrimEnd('/'))</_CopilotNormalizedRegistryUrl>
       <_CopilotDownloadUrl>$(_CopilotNormalizedRegistryUrl)/@github/copilot-$(_CopilotPlatform)/-/copilot-$(_CopilotPlatform)-$(CopilotCliVersion).tgz</_CopilotDownloadUrl>
+      <!-- DownloadFile Timeout is in milliseconds; convert from user-facing seconds -->
+      <_CopilotCliDownloadTimeoutMs>$([System.Convert]::ToInt32($([MSBuild]::Multiply($(CopilotCliDownloadTimeout), 1000))))</_CopilotCliDownloadTimeoutMs>
     </PropertyGroup>
 
     <!-- Delete archive if binary missing (handles partial/corrupted downloads) -->
@@ -75,7 +77,7 @@
     <MakeDir Directories="$(_CopilotCacheDir)" Condition="!Exists('$(_CopilotCliBinaryPath)')" />
     <Message Importance="high" Text="Downloading Copilot CLI $(CopilotCliVersion) for $(_CopilotPlatform)..." Condition="!Exists('$(_CopilotCliBinaryPath)')" />
     <DownloadFile SourceUrl="$(_CopilotDownloadUrl)" DestinationFolder="$(_CopilotCacheDir)" DestinationFileName="copilot.tgz"
-                  Timeout="$([System.Int32]::Parse('$(CopilotCliDownloadTimeout)'))"
+                  Timeout="$(_CopilotCliDownloadTimeoutMs)"
                   Condition="!Exists('$(_CopilotCliBinaryPath)')" />
 
     <!-- Extract using tar (use Windows system tar explicitly to avoid Git bash tar issues) -->

